MaxText/configs/v5p/gpt3_175b/gpt3_175b_base.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# GPT-3 175B Model.
+# This script abstracts away all the shared config between Google's various MLPerf 4.0 submissions, which 
+# achieved high performance on the Training benchmark. See the various different TPU type configs in this 
+# directory to actually run the training.
+
+set -euox pipefail
+
+bash preflight.sh PLATFORM=gke
+sleep 60
```

Comment:
Why is this sleep needed? We don't run sleep in other configs?

Reply:
I based these on the MLPerf scripts we have internally and they had this. I can test without it Monday and see if it works and then update